apps/condo/domains/notification/templates.js

```diff
@@ -172,102 +189,384 @@ async function renderTemplate (transport, message) {
-        } else if (message.lang === 'ru') {
+        }
+
+        if (message.lang === RU_LOCALE) {
             return {
-                subject: 'Код',
-                text: `Код: ${smsCode}`,
+                text: `
+                        Перейдите по сслыке для изменения пароля: ${SERVER_URL}/auth/change-password?token=${token}
+                    `,
             }
         }
     }
+}
+
+const getSmsVerifyCodeMessage = (message) => {
+    const { smsCode } = message.meta
+
+    if (message.lang === EN_LOCALE) {
+        return {
+            subject: 'Verify code',
+            text: `Code: ${smsCode}`,
+        }
+    }
 
-    if (message.type === DEVELOPER_IMPORTANT_NOTE_TYPE) {
-        const { data, type } = message.meta
+    if (message.lang === RU_LOCALE) {
+        return {
+            subject: 'Код',
+            text: `Код: ${smsCode}`,
+        }
+    }
+}
+
+/**
+ * Looks like we shouldn't send such messages through SMS (could be very expensive)
+ * @param message
+ * @returns {{subject: string, text: string}}
+ */
+const getDeveloperImportantNote = (message, transport) => {
+    const { data, type } = message.meta
+
+    if (transport === EMAIL_TRANSPORT) {
         return {
             subject: String(type),
             text: JSON.stringify(data),
         }
     }
+}
 
-    if (message.type === CUSTOMER_IMPORTANT_NOTE_TYPE) {
-        const { data } = message.meta
+const getCustomerImportantNote = (message) => {
+    const { data } = message.meta
 
-        return {
-            subject: 'Новая организация. (СББОЛ)',
-            text: `
+    return {
+        subject: 'Новая организация. (СББОЛ)',
+        text: `
                 Название: ${get(data, ['organization', 'name'])},
                 ИНН: ${get(data, ['organization', 'meta', 'inn'])},
             `,
-        }
     }
+}
 
-    if (message.type === MESSAGE_FORWARDED_TO_SUPPORT) {
-        const { emailFrom, meta } = message
-        const { dv, text, os, appVersion, organizationsData = [] } = meta
+const getTicketAssigneeConnectedMessagePush = (message) => {
+    const { id: notificationId } = message
+    const { ticketId, ticketNumber, userId } = get(message, 'meta', {})
 
-        return {
-            subject: 'Обращение из мобильного приложения',
-            text: `
-                Система: ${os}
-                Версия приложения: ${appVersion}
-                Email: ${emailFrom ? emailFrom : 'не указан'}
-                Сообщение: ${text}
-                УК: ${organizationsData.length === 0 ? 'нет' : organizationsData.map(({ name, inn }) => `
-                  - ${name}. ИНН: ${inn}`).join('')}
-            `,
+    switch (message.lang) {
+        case EN_LOCALE:
+            return {
+                notification: {
+                    title: `Ticket #${ticketNumber}`,
+                    body: `You were assigned as responsible of ticket #${ticketNumber}`,
+                },
+                data: { ticketId, notificationId },
```

Comment:
Could you please collect all these fields in the `message.meta.data`? If done, it will be easier to use templates without building some specific objects, just send `meta.data`.